### DISCUSSION TITLE
How about we remove multiline rows?

### DISCUSSION THREAD

**Discussion by saulpw** [saulpw/visidata#1838] [Polls]
It would make the drawing code a lot simpler, if we removed the `height` and basically everything to do with multi-line rows.  Its only purpose is to make larger cells more visible, but there are other ways we could do this better.  For instance, we could make it easier to show the current cell in the sidebar, or to 'pin' a column in the sidebar (always show that column for the current row).

Anyway, does anyone use this feature a lot?  It's been a thorn in my side ever since I implemented it, and I'm sorely tempted to rip it out, especially as I'm looking at allowing custom display functions that can colorize mid-cell.

**Comment by geekscrapy** [comment]
I have a _lot_ of long lined (and multiline) data! Windows event logs, web logs.
I will say, I like the previous iteration of multilines best. I find I have some rows which are blank as they dont contain much, so I find myself just using it to view one cell (but I wouldn't have known that the row was short without doing a multiline)

I think the pinning is a good way to go, and if we could improve the zl,zh,zk,zj commands in lieu of not having multi-lines that would be amazing

**Reply by saulpw** [reply]
What do you think about changing it so that only the cursor row was multiline (and with a fixed height)?

We could also change the truncation character to show something different in the case of multiple lines or very long strings.  Here are unicode characters with "ellipsis" in the name that we could use:

|name                                    |text  |num   |
|----------------------------------------|------|-----:|
|MONGOLIAN ELLIPSIS                      |᠁     |1801  |
|HORIZONTAL ELLIPSIS                     |…     |2026  |
|VERTICAL ELLIPSIS                       |⋮     |22EE  |
|MIDLINE HORIZONTAL ELLIPSIS             |⋯     |22EF  |
|UP RIGHT DIAGONAL ELLIPSIS              |⋰     |22F0  |
|DOWN RIGHT DIAGONAL ELLIPSIS            |⋱     |22F1  |
|PRESENTATION FORM FOR VERTICAL HORIZONTAL ELLIPSIS|︙     |FE19  |
